<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INLM-MohitR-2.local" ignored="0" name="MicrosoftEdge" tests="6" failures="1" timestamp="2021-10-16T16:57:31 IST" time="45.259" errors="0">
  <testcase name="pageLoad" time="2.331" classname="testcases.AssignmentTest"/>
  <testcase name="allIntegration" time="12.192" classname="testcases.AssignmentTest"/>
  <testcase name="codlessVerification" time="3.545" classname="testcases.AssignmentTest"/>
  <testcase name="learnMore" time="5.494" classname="testcases.AssignmentTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Expected-&amp;gt; Running Automation Tests Using TestingWhiz LambdaTest | LambdaTest
 Actual-&amp;gt; TestingWhiz Integration | LambdaTest
 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Expected-> Running Automation Tests Using TestingWhiz LambdaTest | LambdaTest
 Actual-> TestingWhiz Integration | LambdaTest
 expected [true] but found [false]
at testcases.AssignmentTest.learnMore(AssignmentTest.java:69)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- learnMore -->
  <testcase name="navigateBlog" time="4.706" classname="testcases.AssignmentTest"/>
  <testcase name="communityVerification" time="7.609" classname="testcases.AssignmentTest"/>
</testsuite> <!-- MicrosoftEdge -->
